android/hybridcommon-ui/src/main/java/com/revenuecat/purchases/hybridcommon/ui/PaywallFragment.kt

```diff
@@ -27,14 +27,31 @@ internal class PaywallFragment : Fragment(), PaywallResultHandler {
             paywallResultListener: PaywallResultListener? = null,
             shouldDisplayDismissButton: Boolean? = null,
             offering: Offering? = null,
+        ): PaywallFragment {
+            return newInstance(
+                activity,
+                requiredEntitlementIdentifier,
+                paywallResultListener,
+                shouldDisplayDismissButton,
+                offering?.identifier,
+            )
+        }
+
+        @JvmStatic
+        fun newInstance(
+            activity: FragmentActivity,
+            requiredEntitlementIdentifier: String? = null,
+            paywallResultListener: PaywallResultListener? = null,
+            shouldDisplayDismissButton: Boolean? = null,
+            offeringIdentifier: String? = null,
```

Comment:
I wonder if we should deprecate the old one (and remove it?), I don't think we will be using it since it would be much trickier.